### ISSUE TITLE
Not able to use GPU for prediction! by using pip env instead of Conda env !

### ISSUE THREAD

**Issue by multinucliated** [rolczynski/Automatic-Speech-Recognition#31]
Im not able to use my GPU !  what could be the possible reason or solution ?

1. I created the virtual pip environment(not conda env)
2. ` pip install automatic-speech-recognition `
3. I have manually uninstalled the tensorflow from my virtual environment and installed 
tf gpu 2.2.0 

and Im getting these logs :

2020-07-03 19:01:14.700922: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
2020-07-03 19:01:16.568359: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-07-03 19:01:17.000059: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1561] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce GTX 980M computeCapability: 5.2
coreClock: 1.1265GHz coreCount: 12 deviceMemorySize: 8.00GiB deviceMemoryBandwidth: 149.31GiB/s
2020-07-03 19:01:17.000274: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
2020-07-03 19:01:17.003927: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_10.dll
2020-07-03 19:01:17.007769: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_10.dll
2020-07-03 19:01:17.009106: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_10.dll
2020-07-03 19:01:17.013800: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_10.dll
2020-07-03 19:01:17.016093: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_10.dll
2020-07-03 19:01:17.017343: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudnn64_7.dll'; dlerror: cudnn64_7.dll not found
2020-07-03 19:01:17.017550: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1598] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed
 properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-07-03 19:01:17.018224: I tensorflow/core/platform/cpu_feature_guard.cc:143] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2020-07-03 19:01:17.026847: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x21f43c313f0 initialized for platform Host (this does not guarantee that XLA will be used). Devi
ces:
2020-07-03 19:01:17.027059: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-07-03 19:01:17.027661: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1102] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-07-03 19:01:17.027804: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1108]
2020-07-03 19:01:17.032795: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1561] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce GTX 980M computeCapability: 5.2
coreClock: 1.1265GHz coreCount: 12 deviceMemorySize: 8.00GiB deviceMemoryBandwidth: 149.31GiB/s
2020-07-03 19:01:17.033024: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
2020-07-03 19:01:17.033299: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_10.dll
2020-07-03 19:01:17.033420: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_10.dll
2020-07-03 19:01:17.033786: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_10.dll
2020-07-03 19:01:17.034116: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_10.dll
2020-07-03 19:01:17.034508: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_10.dll
2020-07-03 19:01:17.036010: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudnn64_7.dll'; dlerror: cudnn64_7.dll not found
2020-07-03 19:01:17.036895: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1598] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed
 properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-07-03 19:01:17.101046: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1102] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-07-03 19:01:17.101254: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1108]      0
2020-07-03 19:01:17.101499: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1121] 0:   N
2020-07-03 19:01:17.103170: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x21f53ac4e20 initialized for platform CUDA (this does not guarantee that XLA will be used). Devi
ces:
2020-07-03 19:01:17.103373: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce GTX 980M, Compute Capability 5.2
WARNING:tensorflow:Mixed precision compatibility check (mixed_float16): WARNING
The dtype policy mixed_float16 may run slowly because this machine does not have a GPU.
If you will use compatible GPU(s) not attached to this host, e.g. by running a multi-worker model, you can ignore this warning. This message will only be logged once
Model: "DeepSpeech2"
_________________________________________________________________
Layer (type)                 Output Shape              Param #
=================================================================
X (InputLayer)               [(None, None, 160)]       0
_________________________________________________________________
lambda (Lambda)              (None, None, 160, 1)      0
_________________________________________________________________
conv_1 (Conv2D)              (None, None, 80, 32)      14432
_________________________________________________________________
conv_1_bn (BatchNormalizatio (None, None, 80, 32)      128
_________________________________________________________________
conv_1_relu (ReLU)           (None, None, 80, 32)      0
_________________________________________________________________
conv_2 (Conv2D)              (None, None, 40, 32)      236544
_________________________________________________________________
conv_2_bn (BatchNormalizatio (None, None, 40, 32)      128
_________________________________________________________________
conv_2_relu (ReLU)           (None, None, 40, 32)      0
_________________________________________________________________
reshape (Reshape)            (None, None, 1280)        0
_________________________________________________________________
bidirectional_1 (Bidirection (None, None, 1600)        9993600
_________________________________________________________________
dropout (Dropout)            (None, None, 1600)        0
_________________________________________________________________
bidirectional_2 (Bidirection (None, None, 1600)        11529600
_________________________________________________________________
dropout_1 (Dropout)          (None, None, 1600)        0
_________________________________________________________________
bidirectional_3 (Bidirection (None, None, 1600)        11529600
_________________________________________________________________
dropout_2 (Dropout)          (None, None, 1600)        0
_________________________________________________________________
bidirectional_4 (Bidirection (None, None, 1600)        11529600
_________________________________________________________________
dropout_3 (Dropout)          (None, None, 1600)        0
_________________________________________________________________
bidirectional_5 (Bidirection (None, None, 1600)        11529600
_________________________________________________________________
dense_1 (TimeDistributed)    (None, None, 1600)        2561600
_________________________________________________________________
dense_1_relu (ReLU)          (None, None, 1600)        0
_________________________________________________________________
dropout_4 (Dropout)          (None, None, 1600)        0
_________________________________________________________________
dense_2 (TimeDistributed)    (None, None, 29)          46429
=================================================================
Total params: 58,971,261
Trainable params: 58,971,133
Non-trainable params: 128
_________________________________________________________________
